themes/a6335949-4465-4b71-926c-4a52d34bc9c0/chrome.css

```diff
@@ -1,53 +1,87 @@
 
-findbar {
-	display: flex;
+@-moz-document url-prefix("chrome:") {
```

Comment:
Please, dont add the moz document check

Reply:
Sorry, my mistake. It has been fixed now